provider/testdata/data/json/time_zone/metazone_period@1/ar-EG.json

```diff
@@ -1,1534 +1,1534 @@
 {
   "adalv": {
-    "1970-00-00 00:00": "euce"
+    "0": "euce"
   },
   "aedxb": {
-    "1970-00-00 00:00": "gulf"
+    "0": "gulf"
   },
   "afkbl": {
-    "1970-00-00 00:00": "afgh"
+    "0": "afgh"
   },
   "aganu": {
-    "1970-00-00 00:00": "atla"
+    "0": "atla"
   },
   "aiaxa": {
-    "1970-00-00 00:00": "atla"
+    "0": "atla"
   },
   "altia": {
-    "1970-00-00 00:00": "euce"
+    "0": "euce"
   },
   "amevn": {
-    "1970-00-00 00:00": null,
-    "1991-09-22 20:00": "arme"
+    "0": null,
+    "11427600": "arme"
   },
   "ancur": {
-    "1970-00-00 00:00": "atla"
+    "0": "atla"
   },
   "aolad": {
-    "1970-00-00 00:00": "afwe"
+    "0": "afwe"
   },
   "aqcas": {
-    "1970-00-00 00:00": "auwe",
-    "2009-10-17 18:00": "case",
-    "2010-03-04 15:00": "auwe",
-    "2011-10-27 18:00": "case",
-    "2012-02-21 17:00": "auwe",
-    "2016-10-21 16:00": "case",
-    "2018-03-10 17:00": "auwe",
-    "2018-10-06 20:00": "case",
-    "2019-03-16 16:00": "auwe",
-    "2019-10-03 19:00": "case",
-    "2020-03-07 16:00": "auwe",
-    "2020-10-03 16:01": "case"
+    "0": "auwe",
+    "20931480": "case",
```

Comment:
Thought: I wonder if we should serialize the integer in Postcard and the nice string in JSON.

The way to do this would be to make a new type like `MinutesSinceEpoch`, use it instead of `i32` in the ZeroMap, and give it a custom `Serialize` and `Deserialize` impl.

I don't want you to do it in this PR, but if you think it's a good idea, you can open an issue and do it as a follow-up.

Reply:
What benefits we can get from it apart from type checking and code style?

Reply:
Json file would be more readable, json file API customized.

Serialize as raw date timestamp in Json and deserialize as `MinutesSinceEpoch`.

File an issue for it.